users/views.py

```diff
@@ -0,0 +1,18 @@
+from django.shortcuts import render
+
+from blog.models import PostPage
+from users.models import User
+
+
+def user_profile(request, username):
+    user = User.objects.filter(username__iexact=username)
+    ctx = {}
+    for field in user:
+        ctx['about'] = field.about
+        ctx['first_name'] = field.first_name
+        ctx['last_name'] = field.last_name
```

Comment:
cos tu jest nie tak, dlaczego uzywasz filter jesli chcesz tylko jednego usera? uzywaj get i wtedy nie potrzebujesz petli, wyglada to super brzydko i be, fuj, przepisywac!!!!!!!